### PR TITLE
General Reordering

### PR DESCRIPTION
**Pull request by XylotrupesGideon**
I have reviewed general items and animals and moved a lot of icons to other categories where they fit better.

plants from "General_Items" have been moved to "Plants_Algae"
Animal experiments have been moved from "Animals" to "Procedures"

I fixed one or two spelling mistakes and edited a name to improve searchability.

### CODE REVIEW

File: static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg
Comment (duerrsimon) [inline static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg:1]: Why did you change this illustration and didn't add a new one? I assume the author here is yourself and not DBCLS? 
The old one is clearer and less sketchy.

File: static/icons/cc-by-4.0/Animals/DBCLS/nematode.svg
Comment (duerrsimon) [inline static/icons/cc-by-4.0/Animals/DBCLS/nematode.svg:1]: Same for this one. People might want to use the simple sketch of the nematode. 
The new one is a bit too detailed for a vector image and quite large (for a vector). Vector images should be <150kb. 
Also in this graphic there are many styles from illustrator that might not work in other programs. 
I would suggest simplyfying the illustration a bit, removing the css classes for inline styles and having a seperate nematode-detailed.svg

File: static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg
Comment (XylotrupesGideon) [reply static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg:1]: Oh damn sorry these two images were not commited in the main and I accidentally added them to the commit

File: static/icons/cc-by-4.0/Animals/DBCLS/nematode.svg
Comment (XylotrupesGideon) [reply static/icons/cc-by-4.0/Animals/DBCLS/nematode.svg:1]: same here it was not commited in the main and i commited it on accident

File: static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg
Comment (XylotrupesGideon) [reply static/icons/cc-by-4.0/Animals/DBCLS/zebrafish.svg:1]: these two images were made by someone else

**Comment by XylotrupesGideon** [comment]
Okay I reverted ack to a version that has the simplified and the detailed version of the Zebrafish and Nematode icon.

Disclaimer: I do not know who made the simplified versions of these animals. They were in the DBCLS folder when I forked the repository so I left them in that folder.

**Comment by duerrsimon** [comment]
Thanks for fixing the issues :)
